.github/workflows/release.yaml

```diff
@@ -23,10 +23,10 @@ jobs:
         uses: docker/setup-buildx-action@v1
 
       - name: Log into GitHub Container Registry
-        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${GITHUB_ACTOR,,} --password-stdin
+        run: echo "${{ secrets.CR_PAT }}" | docker login ghcr.io -u ${GITHUB_ACTOR,,} --password-stdin
```

Comment:
Why has the token changed to CR_PAT, what is it?

Reply:
I faced some issues publishing to ghcr.io, then i created another token, granted permissions and tried, then it started working... Will revert.